src/api/libcellml/units.h

```diff
@@ -271,22 +271,66 @@ class LIBCELLML_EXPORT Units: public ImportedEntity
     void getUnit(size_t index, std::string& reference, std::string &prefix, double &exponent, double &multiplier, double &offset) const;
 
     /**
+     * @brief Get the @c unit attributes for the given reference @p reference.
+     *
+     * Get the attributes for the first @c unit that matches the reference @p reference. If
+     * no attributes are set, default attribute values will be returned.
+     *
+     * @overload
+     *
+     * @param reference The @c std::string reference for the @c unit attributes to get.
+     * @param prefix The prefix for this @c unit. Defaults to empty string.
+     * @param exponent The exponent for this @c unit. Defaults to 1.0.
+     * @param multiplier The multiplier for this @c unit. Defaults to 1.0.
+     * @param offset The offset for this @c unit. Defaults to 0.0.
+     */
+    void getUnit(const std::string &reference, std::string &prefix, double &exponent, double &multiplier, double &offset) const;
+
+    /**
+     * @brief Get the @c unit attributes for the given reference @p standardRef.
+     *
+     * Get the attributes for the first @c unit that matches the reference @p standardRef. If
```

Comment:
Ok, I can see that you have added a [test](https://github.com/cellml/libcellml/pull/143/commits/3175d33a20cdb8d76bbe9d4b5c0a98efbb0167fa) for it. I stand corrected.